web/src/read_receipts.ts

```diff
@@ -28,18 +38,23 @@ export function show_user_list(message_id) {
                 $("#read_receipts_modal .modal__content").addClass("compact");
             } else {
                 loading.make_indicator($("#read_receipts_modal .loading_indicator"));
-                channel.get({
+                void channel.get({
                     url: `/json/messages/${message_id}/read_receipts`,
                     success(data) {
-                        const users = data.user_ids.map((id) => {
+                        const clean_data = read_receipts_api_response_schema.parse(data);
```

Comment:
Use `raw_data` and `data`, rather than `data` and `clean_data`.

Reply:
Done.Thanks !!